lib/file-view.coffee

```diff
@@ -19,7 +19,11 @@ class FileView extends HTMLElement
     @fileName.dataset.name = @file.name
     @fileName.dataset.path = @file.path
 
-    @fileName.classList.add(FileIcons.getService().iconClassForPath(@file.path))
+    iconClass = FileIcons.getService().iconClassForPath(@file.path, this)
+    if iconClass
+      unless Array.isArray iconClass
```

Comment:
I think it's safe to assume many authors (particularly those used to working with jQuery) might want to pass multiple classes as a space-separated string. E.g., they wouldn't expect `first second` to throw an error.